snow/consensus/snowball/binary_snowflake.go

```diff
@@ -11,29 +11,37 @@ func newBinarySnowflake(alphaPreference, alphaConfidence, beta, choice int) bina
 	return binarySnowflake{
 		binarySlush:     newBinarySlush(choice),
 		alphaPreference: alphaPreference,
-		alphaConfidence: alphaConfidence,
-		beta:            beta,
+		alphaConfidence: []int{alphaConfidence},
+		beta:            []int{beta},
+		confidence:      make([]int, 1),
```

Comment:
If we are expecting all of these to have the same value, it seems like it might make more sense to have a single slice with a struct.